test/transforms/zscore.jl

```diff
@@ -86,4 +86,14 @@
   @test isapprox(σ[2], 1; atol=1e-6)
   tₒ = revert(T, n, c)
   @test Tables.matrix(t) ≈ Tables.matrix(tₒ)
+  
+  a=rand(Int,3)
+  b=rand(3)
```

Comment:
```suggestion
  a = rand(Int, 10)
  b = rand(10)
```
Slightly increase the number of elements of vectors `a` and `b`.